src/screenControl.py

```diff
@@ -115,7 +118,7 @@ def outputBottom(self):
         (maxy, maxx) = self.screenControl.getScreenDimensions()
         borderY = maxy - 2
         # first output text since we might throw an exception during border
-        usageStr = SHORT_NAV_USAGE if self.mode == SELECT_MODE else SHORT_COMMAND_USAGE
+        usageStr = SHORT_NAV_USAGE if self.mode == SELECT_MODE or X_MODE else SHORT_COMMAND_USAGE
```

Comment:
No switch statements in python, so I used the old dictionary select trick.  I agree it makes it cleaner.